internal/broker/bind_delete.go

```diff
@@ -48,10 +56,19 @@ func (b *UnbindEndpoint) Unbind(ctx context.Context, instanceID, bindingID strin
 		return domain.UnbindSpec{}, apiresponses.NewFailureResponse(fmt.Errorf("failed to get instance %s", instanceID), http.StatusInternalServerError, fmt.Sprintf("failed to get instance %s", instanceID))
 	}
 
-	err = b.bindingsManager.Delete(ctx, instance, bindingID)
+	// if instance does not exist in instances table - remove binding from bindings table and return 410 (unless other error)
```

Comment:
The comment suggests that the next instructions would be checking if instance "exist in instances table". Consider changing the comment.

Reply:
It does not, but for clarity and disambiguation I remove the comment.
This is quotation from the ticket.